rmgpy/rmg/model.py

```diff
@@ -233,6 +233,7 @@ def __init__(self, core=None, edge=None):
         self.kineticsEstimator = 'group additivity'
         self.indexSpeciesDict = {}
         self.saveEdgeSpecies = False
+        self.iteration = 0
```

Comment:
Suggestion, feel free to ignore, perhaps this could be called `iterationCounter` or `iterationNum` for clarity.